docs/runtimes/application.ipynb

```diff
@@ -135,6 +135,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other API Gateway configurations\n",
+    "\n",
+    "There are also other parameters that can be configured when creating an API gateway:"
```

Comment:
"There are additional parameters that can be configured when creating an API gateway:"